lua/darkrp_config/settings.lua

```diff
@@ -69,7 +69,7 @@ GM.Config.deadtalk                      = true
 GM.Config.deadvoice                     = true
 -- deathpov - Enable/disable whether people see their death in first person view.
 GM.Config.deathpov                      = false
--- decalcleaner - Enable/Disable clearing every player's decals.
+-- decalcleaner - Enable/Disable clearing ever player's decals.
```

Comment:
Every* 🙈

Reply:
Oh wow, don't know how I didn't see that. Thanks.